.drone.yml

```diff
@@ -95,6 +95,15 @@ steps:
   - initialize
   - lint-backend
 
+- name: validate-scuemata
+  image: grafana/build-container:1.4.1
+  commands:
+  - go run build.go build-cli
```

Comment:
Isn't grafana-cli already built by the build-backend step?

Reply:
Yeah, didn't see that - changing.